agentos/cli.py

```diff
@@ -131,56 +144,63 @@ def init(dir_names, agent_name, agentos_dir):
     )
 
 
-# TODO - reimplement hz and max_iters
+# TODO - reimplement hz
 @agentos_cmd.command()
 @_option_num_episodes
 @_option_test_every
 @_option_test_num_episodes
 @_option_agent_file
 @_option_agentos_dir
+@_option_max_transitions
 @_option_verbose
 def learn(
     num_episodes,
     test_every,
     test_num_episodes,
     agent_file,
     agentos_dir,
+    max_transitions,
     verbose,
 ):
-    _check_path_exists(agentos_dir)
     agentos.learn(
         num_episodes=num_episodes,
         test_every=test_every,
         test_num_episodes=test_num_episodes,
         agent_file=agent_file,
         agentos_dir=agentos_dir,
+        max_transitions=max_transitions,
         verbose=verbose,
     )
 
 
-# TODO - reimplement hz and max_iters
+# TODO - reimplement hz
 @agentos_cmd.command()
 @_option_num_episodes
 @_option_agent_file
 @_option_agentos_dir
+@_option_max_transitions
 @_option_verbose
-def run(num_episodes, agent_file, agentos_dir, verbose):
+def run(num_episodes, agent_file, agentos_dir, max_transitions, verbose):
```

Comment:
+1 to supporting a `max_transitions` argument. What do you think of making it optional (i.e. have a default value of `None`)? If it is required, then one can't let an agent just run till its done regardless of how long it takes to achieve that, which seems like an option we want the developer to have.